bin/push-tag.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+set -e
+
+# Insist repository is clean
+git diff-index --quiet HEAD
+
+git checkout $1
+git pull origin $1
+git push origin $1
+
+version=$(grep "__version__ = " spacy/about.py)
```

Comment:
This needs to be updated ;-)

Reply:
Oh, sorry, I didn't check whether it was right script. We should redo all these to just have one option...

Reply:
Should be fixed by https://github.com/explosion/spacy-legacy/pull/25 :-)